pan/legacy.pan

```diff
@@ -1,3 +1,12 @@
 declaration template pan/legacy;
 
 type legacy_binary_affirmation_string = string with match(SELF, "^(yes|no)$");
+
+function is_yes_no_true_false = {
+    v = ARGV[0];
+    if (is_boolean(v)) return(true);
+    if (is_string(v) && match(v, "^(yes|no)$")) return(true);
```

Comment:
My intention here is to actively manage the migration:
1. No components will use this type until their Perl code has been made compatible with real booleans.
2. No component will use this type for more than a single release.